src/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 """azplugins."""
 
-from hoomd.azplugins import bond, pair
+from hoomd.azplugins import bond, pair, flow
```

Comment:
Do you have pre-commit setup? I would think ruff would reject this for being out of order.